api/source/service/mysql/STIGService.js

```diff
@@ -949,10 +949,90 @@ exports.deleteStigById = async function(benchmarkId, userObject) {
  * cci String A path parameter that indentifies a CCI
  * returns List
  **/
-exports.getCci = async function(cci, userObject) {
+exports.getCci = async function(cci, inProjection, userObject) {
+  let columns = [
+    'c.cci', 
+    'c.status', 
+    'c.publishdate', 
+    'c.contributor', 
+    'c.type', 
+    'c.definition'
+  ]
+
+  let joins = [
+    'cci c '
+  ]
+  
+  let predicates = {
+    statements: [],
+    binds: []
+  }
+  
+  // PREDICATES
+  predicates.statements.push('c.cci = ?')
+  predicates.binds.push(cci)
+
+  if ( inProjection && inProjection.includes('emassAp') ) {
+    columns.push(`case when c.apAcronym is null then null else json_object("apAcronym", c.apAcronym, "implementation", c.implementation, "assessmentProcedure", c.assessmentProcedure) END  as "emassAp"`)
+  }
+
+  if ( inProjection && inProjection.includes('references') ) {
+    columns.push(`(select 
+      coalesce
+      (
+        (
+          select json_arrayagg (json_object(
```

Comment:
Please include the other properties from the OAS definition:
```
        "version": "string",
        "location": "string",
        "indexDisa": "string",
        "textRefNist": "string",
        "parentControl": "string"
```
these are all in `crm` which is handy!

Reply:
added